client/components/clubs/ClubList.tsx

```diff
@@ -1,39 +1,26 @@
-import { SearchItem, SearchResult, useSearch } from "../hooks";
-import { ClubCard } from "./ClubCard";
-import { Id, ClubInfo } from "../../context";
+import { useSearch } from '../hooks'
+import { ClubCard } from './ClubCard'
+import { Id, Club } from '../../context'
+import { useMemo } from 'react'
 
 interface ClubListProps {
-  clubListData: Map<Id, ClubInfo>;
+  clubs: Map<Id, Club>
 }
 
-const clubsList = (filteredClubs: SearchResult) => {
-  return filteredClubs.map((club: SearchItem | { item: SearchItem }) => {
-    if ("item" in club) {
-      club = club.item as SearchItem;
-    }
```

Comment:
What happened to this logic?

Reply:
It's because the changes to searchResult standardized its return type to always be an array of the same type as what was passed in:

```
const searchResult: Array<T> = searchValue
  ? fuse.search(searchValue).map(({ item }: Fuse.FuseResult<T>) => item)
  : items
```

Now, when we take the results from the search hook, they are always `Club[]` rather than `(FuseThingy | Club)[]`, so there's no need to adjust based on the type of the array element